packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -10,6 +10,11 @@ UserMessages get userMessages => context.get<UserMessages>();
 
 /// Class containing message strings that can be produced by Flutter tools.
 class UserMessages {
+  // Common messages.
+  String get flutterToolBugInstructions =>
+      'Please file a bug at https://github.com/flutter/flutter/issues.';
```

Comment:
I like I like the "Please let us know at" text better, I say "file" all the time but I don't think everyone uses that verb.

Reply:
How about "Please report"?